### DISCUSSION TITLE
How do I stop creating a slug and just leave the tag text ?

### DISCUSSION THREAD

**Discussion by proxima812** [Thinkmill/keystatic#663] [Q&A]
```
tags: collection({
label: "Tags",
path: 'src/content/tags/*',
slugField: 'name',
format: {
data: 'yaml',
content Field: 'mark doc',
},
schema: {
mark doc: fields.empty Document(),
name: fields.slug({ name: {label: 'Tag name'} }),
},
```

The problem is that I need to output the tag value without slug. How to do it. I have posts with the slug value coming out. example: 12-steps. And the name is "12 steps"

**Comment by simonswiss** [comment]
Hey!

What you have in place _should_ work: the `name` field can be a text string like `12 steps`, and a `12-steps` slug will be generated from that string (with the ability to manually change it).

Perhaps you're trying to use `await reader.collections.tags.list()` from the [Reader API](https://keystatic.com/docs/reader-api)? This will only return an array of `slugs`.

But if you get all tag entries with `await reader.collections.tags.all()`, you'll also have access to the `entry` data for each tag.

![CleanShot 2023-10-16 at 15 00 02@2x](https://github.com/Thinkmill/keystatic/assets/485747/3a151371-6954-414d-abfc-b66f715d1ae8)

In there,  you should be able to read the full `name` string with `entry.name`.

You can see an example of this in the Keystatic website's blog:

First, we [get all the blog entries](https://github.com/Thinkmill/keystatic/blob/main/docs/src/app/(public)/blog/page.tsx#L10) with `reader.collections.blog.all()`.

Then, we [output the Title field](https://github.com/Thinkmill/keystatic/blob/main/docs/src/app/(public)/blog/page.tsx#L56) (which is [used to generate the `slug`](https://github.com/Thinkmill/keystatic/blob/main/docs/keystatic.config.tsx#L115-L130)) with `entry.title`.

Hope it helps! Happy to help further if you have some code to share.
